doxia-core/src/main/java/org/apache/maven/doxia/parser/AbstractParser.java

```diff
@@ -37,6 +29,13 @@
 import org.apache.maven.doxia.sink.Sink;
 import org.codehaus.plexus.component.annotations.Requirement;
 
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.io.StringReader;
+import java.util.Properties;
+
```

Comment:
is it possible to avoid such reformatting of imports, please?

Reply:
It's by Intellij, I'll restore it in a commit. I have noted many other things that even with the style configured as described in https://maven.apache.org/developers/conventions/code.html#intellij-idea. For example exception below the method definition are indented at 8 spaces while the original code is at 4.
I try to keep some things that I see, but some may slipt, sorry.

Reply:
It's by Intellij, I'll restore it in a commit. I have noted many other things that even with the style configured as described in https://maven.apache.org/developers/conventions/code.html#intellij-idea. For example exception below the method definition are indented at 8 spaces while the original code is at 4.
I try to keep some things that I see, but some may slipt, sorry.